isis/tests/FunctionalTestsHyb2Pds4Gen.cpp

```diff
@@ -0,0 +1,119 @@
+#include <QTextStream>
+#include <QStringList>
+#include <QFile>
+#include <QDomDocument>
+
+#include "hyb2pds4gen.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "SpecialPixel.h"
+#include "ProcessExportPds4.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/hyb2pds4gen.xml").expanded();
+
+TEST_F(Hayabusa2OncW2Cube, FunctionalTestHyb2Pds4GenDefault) {
+  setInstrument("-37100", "ONC-T", "HAYABUSA-2");
+
+  PvlGroup newBandGroup; 
+  std::istringstream  bbstr(R"(
+    Group = BandBin
+      Name         = W
+      Center       = 700
+      Width        = 70
+      Unit         = Nanometers
+      FilterNumber = 4
+    End_Group
+  )");  
+  bbstr >> newBandGroup; 
+
+  PvlGroup &bb = testCube->label()->findObject("IsisCube").findGroup("BandBin");
+  bb = newBandGroup; 
+   
+  QString fileName = testCube->fileName();
+  delete testCube;
+  
+  testCube = new Cube(fileName, "rw");
+
+  QVector<QString> args = {"to=/tmp/output", "PDS4LOGICALIDENTIFIER=Whatever"};
+  UserInterface options(APP_XML, args);
+  
+  hyb2pds4gen(testCube, options);
+  
+  QDomDocument pds4lab;
+
+  QFile f("/tmp/output.xml");
```

Comment:
Won't these collide if we have more than one test running?

Reply:
oof, I made that local for testing purposes and forgot to revert, good catch, my bad.